src/Parser.php

```diff
@@ -14,8 +14,13 @@ public function parse(string $filename)
         foreach ($lines as $line) {
             $line = trim($line);
             if ($line !== '') {
-                [$url, $status] = preg_split('/[\s]+/', $line);
-                $domains[] = new Domain($url, $status);
+                [$url, $status, $content] = preg_split('/[\s]+/', $line);
```

Comment:
Moved to `json` - still feels fairly simple and means new things are *way* easier to add in the future :)